WorkloadManagementSystem/Client/JobReport.py

```diff
@@ -35,18 +35,15 @@ def setJob(self, jobID):
     """
     self.jobID = jobID
 
-  def setJobStatus(self, status='', minor='', application='', sendFlag=True):
+  def setJobStatus(self, status='', minorStatus='', applicationStatus='', sendFlag=True):
```

Comment:
Isn't this a breaking API change after v7r1 has already been released? At the very least I think this should:
```suggestion
  def setJobStatus(self, status='', minorStatus='', applicationStatus='', sendFlag=True, minor=None):
    if minorStatus and minor:
      # Raise an error
    if minor is not None:
      # Show a warning
      minorStatus = minorStatus or minor
```
And then remove `status` in v7r2

Reply:
Sorry, but even if v7r1 has been released, it is broken and thus unusable IMHO...
Anyway most of the calls were using positional arguments and not the keys, thus this won't help... 
B.t.w. all calls have been fixed of course and I doubt there is any call from within an extension (at least not in LHCb AFAIS)

Reply:
Anyway, it doesn't harm, I'll do it then!